### PR TITLE
SP-1, remove deleted configuration from docs

### PR DESCRIPTION
**Pull request by rbohl**
cc @lipusz

### CODE REVIEW

**Comment by rbohl** [comment]
@sez11a going to update this PR in a few minutes (just warning you in case you were going to merge it soon)

**Comment by rbohl** [comment]
Updated, @sez11a 
cc @lipusz

File: en/user/articles-dxp/120-search/13-result-rankings.markdown [hunk 1]
```diff
@@ -93,13 +92,10 @@ To create a new Result Rankings Alias:
...
     worry, you can add more later) in the _Search Query_ field. 
 
-    To limit the rankings intervention to just one index's results, enter the
-    index name in the Index Name field.
-
     Click _Customize Results_.
 
 A search query is executed. The results are displayed and the tools for pinning,
-hiding, re-ordering, and adding results are made available. First, consider
+hiding, and adding results are made available. Re-ordering First, consider
```
Comment (sez11a) [inline en/user/articles-dxp/120-search/13-result-rankings.markdown]: I don't get this change, @rbohl .

**Comment by sez11a** [comment]
Left a comment, @rbohl

File: en/user/articles-dxp/120-search/13-result-rankings.markdown (hunk 1 above)
Comment (rbohl) [reply en/user/articles-dxp/120-search/13-result-rankings.markdown]: It was the beginning of a very promising sentence, whose promise was never fulfilled.

**Comment by rbohl** [comment]
Commit amended to finish sentence

**Pull request merged**